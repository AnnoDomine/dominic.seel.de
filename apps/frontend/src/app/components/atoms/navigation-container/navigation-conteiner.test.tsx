import { describe, it, expect } from "vitest"
import { render, screen } from "@testing-library/react"
import NavigationContainer from "./navigation-container"

describe("NavigationContainer", () => {
    it("renders without crashing", () => {
        render(<NavigationContainer data-testid="navigation-container">Content</NavigationContainer>)
        const navigationContainer = screen.getByTestId("navigation-container")
        expect(navigationContainer).toMatchSnapshot()
    })
})
