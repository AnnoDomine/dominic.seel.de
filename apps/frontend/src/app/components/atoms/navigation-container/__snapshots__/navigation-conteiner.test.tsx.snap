// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NavigationContainer > renders without crashing 1`] = `
<header
  class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-1m7vu0j-MuiPaper-root-MuiAppBar-root"
  data-testid="navigation-container"
  style="--Paper-shadow: 0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);"
>
  Content
</header>
`;
